src/index.html

```diff
@@ -3,7 +3,6 @@
   <head>
     <meta charset="utf-8">
     <title>Fresh Picks</title>
-    <link rel="stylesheet" type="text/css" href="../styles.css">
```

Comment:
Nice get that sneaky little link tag outta here, not needed with webpack. Good catch!